vhdbuilder/packer/list-images.ps1

```diff
@@ -10,7 +10,7 @@ $windowsImageVersion = $env:WindowsImageVersion
 
 $ErrorActionPreference = "Stop"
 
-$imageBomJsonFilePath = "c:\image-bom.json"
+$imageBomJsonFilePath = "c:\image-list.json"
```

Comment:
This file will exist inside every VHD. I think that we should not change this file name to keep it to be consistent with Linux. We only need to rename it before uploading it to artifacts.

Reply:
Thanks for the comment. Just find it and the empty file of image-list.json in artifacts should be caused by this change.